### ISSUE TITLE
Debug message about background images are out of bounds

### ISSUE THREAD

**Issue by Brianum** [EasyRPG/Player#1223]
Thank you for supporting EasyRPG Player.
Before creating an issue, please verify that your bug still occurs in the "continuous builds" of the Player.
You can get them at the bottom of our download section: https://easyrpg.org/player/downloads

---
Please fill in the following fields before submitting an issue:
#### Player platform:
Windows64

#### Attach files (as a .zip archive or link them)
 - A savegame next to the problem (if it is not at the beginning of the game)
 - The `easyrpg_log.txt` log file (you can find it in the game directory)

[2017-07-18 13:41:48] Debug: Image size out of bounds: Backdrop/battleeternal18 (320x160 < 320x240 < 320x160)

#### Describe the issue in detail and how to reproduce it:
Debug message appears saying background image is out of bounds, despite RPG Maker 2003 games have larger battle backdrops (320x240) compared to RPG Maker 2000 (320x160)

**Comment by carstene1ns** [comment]
Thanks, while your point is valid and needs to be fixed, we think these messages are really annoying anyway, so we consider removing them altogether (or at least not showing them by default).

**Comment by Brianum** [comment]
The messages indeed pointed me at one Backdrop that is too big, so my expectation would be making the bounds dependent on the engine version.

**Comment by fdelapena** [comment]
Untested patch:
```diff
diff --git a/src/cache.cpp b/src/cache.cpp
index f035e58f..1838cc5f 100644
--- a/src/cache.cpp
+++ b/src/cache.cpp
@@ -150,7 +150,7 @@ namespace {
                int min_height, max_height;
                std::function<BitmapRef()> dummy_renderer;
        } const spec[] = {
-               { "Backdrop", false, 320, 320, 160, 160, backdrop_dummy_func},
+               { "Backdrop", false, 320, 320, 160, Player::IsRPG2k() ? 160 : 240, backdrop_dummy_func},
                { "Battle", true, 480, 480, 96, 480, battle_dummy_func },
                { "CharSet", true, 288, 288, 256, 256, charset_dummy_func },
                { "ChipSet", true, 480, 480, 256, 256, chipset_dummy_func },
```

**Comment by carstene1ns** [comment]
Not tested, but this patch will likely not work.

**Comment by Ghabry** [comment]
yepp because that array is constant. Probably won't even compile because Player::IsRPG2k is not a constexpr.

**Comment by fdelapena** [comment]
It builds without warnings for me, but not tested with games yet.

**Comment by carstene1ns** [comment]
While this is not really fixed, that warning vanished since #1239.
